### PR TITLE
[DO NOT MERGE] test downstream CI ignores

### PR DESCRIPTION
**Pull request by davidcheung**
DO NOT MERGE, using PR to test if ignore downstream worked
related to #3000

### CODE REVIEW

**Comment by davidcheung** [comment]
this did not work, perhaps https://github.com/strongloop/loopback/pull/3000/commits/a759286330fbf3fcbf2c83957ba44b047b856b1f needs to be merged into master for Jenkins to pick it up 
cc @pthieu @superkhau

**Comment by superkhau** [comment]
@davidcheung Yes, I believe that has to be merged into master for it to work. @rmg Can you chime in here? @davidcheung is trying to get downstream ignores to work on the 2.x branch. Is it not good enough to modify the package.json on the 2.x branch?

**Comment by rmg** [comment]
It has to be merged into the destination branch. Each monitored branch is a separate project.

**Comment by davidcheung** [comment]
@rmg the commit is already merged into destination branch (2.x) https://github.com/strongloop/loopback/pull/3000/commits/a759286330fbf3fcbf2c83957ba44b047b856b1f
is the package.json name the correct name to use? 
as the jenkins job shows it with the branch name like `dashboard-controller~master` vs `dashboard-controller`

**Comment by rmg** [comment]
@davidcheung the `package.json` is correct, but the repo scanner was disabled for the last few days due to a plugin rollback. I fixed it earlier today and the jobs were regenerated about 90 minutes ago. Rebasing this PR should do the correct thing now.

File: package.json [hunk 1]
```diff
@@ -113,7 +113,8 @@
       "debug": "*,-mocha:*,-eslint:*",
       "downstreamIgnoreList" : [
         "dashboard-controller",
-        "gateway-director-management-interface"
+        "gateway-director-management-interface",
```
Comment (pthieu) [inline package.json]: Testing new GH review

**Comment by davidcheung** [comment]
@slnode test please

**Comment by davidcheung** [comment]
Looks like it still doesnt ignore, 
>- [cis-jenkins] downstream: dashboard-controller@develop — Failed! (e9d7396)
>- [cis-jenkins] downstream: gateway-director-management-interface@master — Failed! (e9d7396)

**Comment by rmg** [comment]
The ignore list was added incorrectly so it is not actually being used. See https://github.com/strongloop/loopback/pull/3000#issuecomment-270188007

**Comment by davidcheung** [comment]
@slnode test please 
looks like some network issue https://cis-jenkins.swg-devops.com/job/npr/job/loopback~2.x/165/
trying again, then should work and can close.

**Comment by davidcheung** [comment]
test success, closing

**Pull request closed without merging**